canvas_cli/apps/plugin/plugin.py

```diff
@@ -204,8 +204,8 @@ def enable(
         print(f"Failed to connect to {host}")
         raise typer.Exit(1)
 
-    if r.status_code == requests.codes.no_content:
-        print(r.text)
+    if r.status_code == requests.codes.ok:
```

Comment:
I think you can actually do:
```suggestion
    if r.status_code.ok:
```

I would test first

Reply:
it didn't work: 
`AttributeError: 'int' object has no attribute 'ok'`

Reply:
there is an `r.ok` that returns a boolean though!